src/Cms/Picker.php

```diff
@@ -98,7 +98,7 @@ public function itemsToArray($items = null): array
             if (empty($this->options['map']) === false) {
                 $result[] = $this->options['map']($item);
             } else {
-                $result[] = $item->panelPickerData([
+                $result[] = $item->panel()->pickerData([
```

Comment:
If I understand it correctly, `$item` can in theory be any model. If this is the case, the `Panel\Model` class should have an abstract definition of `pickerData` to avoid failures down the road.

Reply:
I agree that this would help. But I would handle that in another PR.

Reply:
I've opened an issue here so that we don't forget about it: https://github.com/getkirby/kirby/issues/3340